plugins/devloop/scripts/devloop-notify.sh

```
@@ -103,18 +103,18 @@ fi
-if [[ -n "${DEV_LOOP_TRANSCRIPT_PATH:-}" ]]; then
-  message="$message | transcript: ${DEV_LOOP_TRANSCRIPT_PATH}"
+if [[ -n "${DEVLOOP_TRANSCRIPT_PATH:-}" ]]; then
+  message="$message | transcript: ${DEVLOOP_TRANSCRIPT_PATH}"
 fi
-export DEV_LOOP_MESSAGE="$message"
+export DEVLOOP_MESSAGE="$message"
 # Provide a pre-quoted version for safer use in templates.
 # Declare and assign separately to avoid masking exit status (SC2155).
-DEV_LOOP_MESSAGE_QUOTED="'$(echo "$message" | sed "s/'/'\\\\''/g")'"
-export DEV_LOOP_MESSAGE_QUOTED
+DEVLOOP_MESSAGE_QUOTED="'$(echo "$message" | sed "s/'/'\\\\''/g")'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)